LandSandBoat/server
# Boyahda tree droplist fixes

Co-authored-by: metal_agentx@yahoo.com

Metal did the code adjustments, I did the testing and this PR.

Fixes https://github.com/LandSandBoat/server/issues/975



**_I affirm:_**

## What does this pull request do?

mob_droplist.sql
- Adds to Old Goobbue:
Boyahda Moss (24%), Tree Cuttings (15%), Goobbue Humus (50%), Boyahda Coffer Key (10%)
- Adds to Korrigan:
Two Leaf Mandragora Bud (10%), Jug of Humus (10%), Four Leaf Korrigan Bud (10%), Two Leaf Mandragora Bud (Despoil)
- Adds to Boyahda Sapling:
Treant bulb (24%), Fruit seeds (10%)
- Drop rate for Earth Cluster off of Mourning Crawler changed to 0% (until mog tablets are implemented)
- Drop rate for Wind Cluster off of Snaggletooth Peapuk changed to 0% (until mog tablets are implemented)
- Drop rate for Water Cluster off of Viseclaw changed to 0% (until mog tablets are implemented)
- I also fixed tab spacings for comments of other lines at the end of mob_droplist.sql to be in line with other drop groups (since it was bugging me and that's where I was at in the file for the above new drop groups)


mob_groups.sql
- Adds new drop group (3228) for Old Goobbue
- Adds new drop group (3229) for Korrigan
- Adds new drop group (3230) for Boyahda Sapling
- Removes drop group for Blood Ball (no credible source for drop rates found)

All drop rates per FFXIDB (or per Claywar in the case of the elemental cluster changes)

## Steps to test these changes

- Cherry-pick this PR (or wait until it's merged and resync your server with LSB Base branch) and notice that Old Goobues, Korrigans, and Boyahda Saplings now drop what they should.
- Also notice that the elemental clusters don't drop off of mobs they shouldn't.
- Also notice that Blood Balls no longer have drops of Blood Bats.
